panel/viewable.py

```diff
@@ -421,6 +440,18 @@ def get_root(self, doc=None, comm=None):
         state._views[ref] = (self, root, doc, comm)
         return root
 
+    def ipywidget(self):
```

Comment:
I think I'd prefer this as a top-level function like `pn.ipy(panel_obj)` for brevity or `pn.ipywidget(panel_obj)` if it's important to be explicit, so that it reads as "wrap the given object as an ipywidget.